### PR TITLE
Remove extra space before semicolon and remove extra comma in php, phtml and js files

### PR DESCRIPTION
**Pull request by tejash-wagento**
### Description

Remove extra space before semicolon and remove extra comma in php, phtml and js files

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6580](https://jira.corp.magento.com/browse/ENGCOM-6580) has been created to process this Pull Request
:eight_spoked_asterisk: @dmytro-ch, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/Quote/Test/Unit/Model/Product/Plugin/UpdateQuoteItemsTest.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function aroundUpdateDataProvider()
             [10, 20, 'once'],
             [null, 10, 'never'],
             [10, 10, 'never'],
-            [10, 10, 'once', true],
```
Comment (orlangur) [inline app/code/Magento/Quote/Test/Unit/Model/Product/Plugin/UpdateQuoteItemsTest.php]: This change is definitely not needed

File: app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     define('analyticsPopupConfig', function () {
         return {
             analyticsVisible: <?= $block->getNotification()->isAnalyticsVisible() ? 1 : 0; ?>,
-            releaseVisible: <?= $block->getNotification()->isReleaseVisible() ? 1 : 0; ?>,
```
Comment (orlangur) [inline app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml]: This change is not needed if it works with trailing comma.

**Review by orlangur — CHANGES_REQUESTED** [review]

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
@tejash-wagento, could you please also check and fix the static test failures?

File: app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml (hunk 2 above)
Comment (hostep) [reply app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml]: I agree with @orlangur here, keeping comma's in javascript objects or php arrays is a good thing, it gives a clearer diff in git when one or multiple lines are added if a comma already existed on the line above the new line.

For javascript, historically this was an issue in older Internet Explorer versions, they couldn't handle that, but according to [this comment](https://stackoverflow.com/a/19791620/145829) it was fixed in IE9 and higher and Magento only supports IE11, so we should be fine.

PHP is more and more encouraging this kind of syntax btw, PHP 7.3 got a new feature where arguments in function/method calls can now also have a trailing comma: https://www.php.net/manual/en/migration73.new-features.php#migration73.new-features.core.trailing-commas

File: app/code/Magento/Quote/Test/Unit/Model/Product/Plugin/UpdateQuoteItemsTest.php (hunk 1 above)
Comment (tejash-wagento) [reply app/code/Magento/Quote/Test/Unit/Model/Product/Plugin/UpdateQuoteItemsTest.php]: added comma after array

File: app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml (hunk 2 above)
Comment (tejash-wagento) [reply app/code/Magento/AdminAnalytics/view/adminhtml/templates/notification.phtml]: @orlangur, comma added after array, 
Thanks for suggestion.

**Comment by tejash-wagento** [comment]
@dmytro-ch, issue fixed

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6580](https://jira.corp.magento.com/browse/ENGCOM-6580) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Pull request merged**